backend/src/events/controllers.ts

```diff
@@ -100,12 +100,20 @@ const getEvents = async (
   let whereDict: { [key: string]: any } = {};
   let includeDict: { [key: string]: any } = {};
 
-  // Handles GET /events?upcoming=true
-  if (filter.upcoming === "true") {
-    const dateTime = new Date();
-    whereDict["startDate"] = {
-      gt: dateTime,
-    };
+  // Handles GET /events?date=upcoming and GET /events?date=past
+  // TODO: Investigate creating events that occur in a few minutes into the future
+  const dateTime = new Date();
+  switch (filter.date) {
+    case "upcoming":
+      whereDict["startDate"] = {
+        gt: dateTime,
+      };
+      break;
+    case "past":
+      whereDict["endDate"] = {
+        lt: dateTime,
+      };
+      break;
```

Comment:
Replaced `GET /events?upcoming=true` and `GET /events?upcoming=false` with `GET /events?date=upcoming` and `GET /events?date=past` for cleaner API syntax